### PR TITLE
Added orca log file reading capability

### PR DESCRIPTION
**Pull request by evohringer**
This adds a orca log output file reader which reads in the SCF energy and atomic numbers and coordinates.
This new function is tested in test_orca for which a new log file was created in the data directory.
I also changed the horton-convert script.

Please provide feedback since this would be a draft for other QC output file readers.

Refs: https://github.com/theochem/iodata/issues/43

### CODE REVIEW

**Comment by tovrstra** [comment]
I'm going to copy-paste a few of the test outputs for convenience, so you can see what it complains about:

```
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
###                               pycodestyle                                ###
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
USING              : 2.5.0
RUNNING            : pycodestyle iodata/orca.py iodata/test/test_orca.py --config=.pycodestylerc
68:24     iodata/orca.py  E225 missing whitespace around operator
71:13     iodata/orca.py  E265 block comment should start with '# '
79:1      iodata/orca.py  E302 expected 2 blank lines, found 1
103:1     iodata/orca.py  E303 too many blank lines (3)
129:22    iodata/orca.py  E231 missing whitespace after ','
130:22    iodata/orca.py  E231 missing whitespace after ','
131:22    iodata/orca.py  E231 missing whitespace after ','
137:22    iodata/orca.py  E231 missing whitespace after ','
138:22    iodata/orca.py  E231 missing whitespace after ','
139:22    iodata/orca.py  E231 missing whitespace after ','
50:1      iodata/test/test_orca.py  E302 expected 2 blank lines, found 1
56:1      iodata/test/test_orca.py  E302 expected 2 blank lines, found 1
68:1      iodata/test/test_orca.py  E302 expected 2 blank lines, found 1
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
###                                pydocstyle                                ###
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
USING              : 3.0.0
RUNNING            : pydocstyle iodata/orca.py iodata/test/test_orca.py
39:-      iodata/orca.py  D414 Section has no content ('Notes')
80:-      iodata/orca.py  D208 Docstring is over-indented
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
###                                whitespace                                ###
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
115:-     iodata/test/data/water_orca.out  trailing whitespace
201:-     iodata/test/data/water_orca.out  trailing whitespace
290:-     iodata/test/data/water_orca.out  trailing whitespace
291:-     iodata/test/data/water_orca.out  trailing whitespace
339:-     iodata/test/data/water_orca.out  trailing whitespace
340:-     iodata/test/data/water_orca.out  trailing whitespace
341:-     iodata/test/data/water_orca.out  trailing whitespace
342:-     iodata/test/data/water_orca.out  trailing whitespace
343:-     iodata/test/data/water_orca.out  trailing whitespace
344:-     iodata/test/data/water_orca.out  trailing whitespace
345:-     iodata/test/data/water_orca.out  trailing whitespace
346:-     iodata/test/data/water_orca.out  trailing whitespace
410:-     iodata/test/data/water_orca.out  trailing whitespace
416:-     iodata/test/data/water_orca.out  trailing whitespace
```

and 

```
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
###                                  pylint                                  ###
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
USING              : pylint 2.3.1astroid 2.2.4
RUNNING            : pylint iodata/orca.py iodata/test/test_orca.py --jobs=2 --output-format=json
29:-      iodata/orca.py  unused-import Unused set_four_index_element imported from utils
68:22     iodata/orca.py  bad-whitespace Exactly one space required after assignment
                words =line.split()
                      ^
129:21    iodata/orca.py  bad-whitespace Exactly one space required after comma
        coordinates[i,0] = float(words[5])
                     ^
130:21    iodata/orca.py  bad-whitespace Exactly one space required after comma
        coordinates[i,1] = float(words[6])
                     ^
131:21    iodata/orca.py  bad-whitespace Exactly one space required after comma
        coordinates[i,2] = float(words[7])
                     ^
134:4     iodata/orca.py  unreachable Unreachable code
137:21    iodata/orca.py  bad-whitespace Exactly one space required after comma
        coordinates[i,0] = float(words[5])
                     ^
138:21    iodata/orca.py  bad-whitespace Exactly one space required after comma
        coordinates[i,1] = float(words[6])
                     ^
139:21    iodata/orca.py  bad-whitespace Exactly one space required after comma
        coordinates[i,2] = float(words[7])
                     ^
25:-      iodata/test/test_orca.py  unused-import Unused import os
56:-      iodata/test/test_orca.py  missing-docstring Missing function docstring
72:-      iodata/test/test_orca.py  superfluous-parens Unnecessary parens after 'assert' keyword
```

**Comment by tovrstra** [comment]
@evohringer Can you also add an exclusion for data files to the whitespace linter? This can be done in the following place:
https://github.com/theochem/iodata/blob/master/.cardboardlint.yml#L11

The following should make more sense:

```yaml
- whitespace:
    filefilter: ['- iodata/test/data/*']
```

**Comment by evohringer** [comment]
I corrected everything in the forked repo. Should I do a new pull request?

**Comment by tovrstra** [comment]
The tests bumped into a few more things. See https://travis-ci.org/theochem/iodata/jobs/503215662#L912

```
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
###                                  pylint                                  ###
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
USING              : pylint 2.3.1astroid 2.2.4
RUNNING            : pylint iodata/orca.py iodata/test/test_orca.py --jobs=2 --output-format=json
29:-      iodata/orca.py  unused-import Unused set_four_index_element imported from utils
131:4     iodata/orca.py  unreachable Unreachable code
25:-      iodata/test/test_orca.py  unused-import Unused import os
58:-      iodata/test/test_orca.py  missing-docstring Missing function docstring
75:-      iodata/test/test_orca.py  superfluous-parens Unnecessary parens after 'assert' keyword
```

**Comment by evohringer** [comment]
Sorry @tovrstra to bother with this. Is there a way I can check this before making a pull request?

**Comment by tovrstra** [comment]
Good point. Yes, but it is not as easy as it should be. (I'm doing something about it.) You can run the following:

```bash
pip install --upgrade git+https://github.com/theochem/cardboardlint.git@master#egg=cardboardlint
pip install --upgrade pylint codecov pycodestyle pydocstyle
python setup.py build_ext -i
cardboardlinter -r origin/master
```

The last line will do the actual work, with the first two lines installing software, which you may already have. The third line does an in-place build, which is needed to keep pylint happy with the Python extension in iodata.

Can you give this a try? If something breaks, please let me know.

**Comment by evohringer** [comment]
It works , great. Thanks.


 but it does not print out any errors:
`RUNNING            : git diff -U0 origin/master --relative
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
###                                  import                                  ###
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
###                                namespace                                 ###
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
###                                  pylint                                  ###
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
USING              : pylint 1.6.4, astroid 2.2.4

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
###                               pycodestyle                                ###
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
USING              : 2.5.0


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
###                                pydocstyle                                ###
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
USING              : 3.0.0


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
###                                whitespace                                ###
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

**Comment by tovrstra** [comment]
That probably is probably because you used the `master` branch (and also `origin/master`) for your development. Do you still have a reference in your repo to the master branch of `theochem/iodata`? That would be needed to get the same test output locally.

It is in general a good idea to use the master branch of cloned repos only to follow up the upstream commits. For clarity, it is recommended to make a new "feature" branch in which you make your changes. This way, you can always easily compare with what you started from, or update the master branch with new upstream developments. 

This is a quick summary of that workflow, assuming you would start from scratch, just to sketch the idea:

```bash
# Clone the primary repo first
git clone git@github.com:theochem/iodata.git
# Now `origin` refers to theochem/iodata
# Add your fork as the second remote (after making the fork on github.com)
cd iodata
git remote add qcmm git@github.com:qcmm/iodata.git
# Make a feature branch, in which a new feature will be developed
git checkout -b orca
# After some changes commit these:
git add ...
git commit ...
# Push the feature branch to  your repo
git push qcmm feature
# A link will be printed on your terminal to make a pull request
# If needed make more commits in the feature branch and push them to qcmm/iodata.
# The pull request will be updated automatically.
```

With this setup `origin/master` still refers to the one from `theochem/iodata`.

You can check your current remotes as follows, with the output I typically get as comments:

```bash
git remote show
# origin
# tovrstra
git remote show origin
#* remote origin
#  Fetch URL: git@github.com:theochem/iodata.git
#  Push  URL: git@github.com:theochem/iodata.git
#  HEAD branch: master
#  Remote branches:
#    master            tracked
#  Local branch configured for 'git pull':
#    master rebases onto remote master
#  Local ref configured for 'git push':
#    master pushes to master (up to date)
git remote show tovrstra
#* remote tovrstra
#  Fetch URL: git@github.com:tovrstra/iodata.git
#  Push  URL: git@github.com:tovrstra/iodata.git
#  HEAD branch: master
#  Remote branches:
#    master  new (next fetch will store in remotes/tovrstra)
#  Local refs configured for 'git push':
#    master pushes to master (fast-forwardable)
```
(I removed some branches for clarity.)
With this setup, I only pull commits from `origin` (short name for `theochem/iodata`), never push to it, so I don't even need write access to it. It can pull and push with the second remote, i.e. `tovrstra`. I guess you now have

```bash
git remote show origin
#* remote origin
#  Fetch URL: git://github.com/QCMM/iodata.git
#  Push  URL: git://github.com/QCMM/iodata.git
#  HEAD branch: master
#  Remote branches:
#    master            tracked
#  Local ref configured for 'git push':
#    master pushes to master (local out of date)
```

The following commands will show how to switch in a new clone of your fork. A verbose command prompt helps a lot to see on which branch you are working. (See e.g. https://gist.github.com/kevin-smets/8568070 for an OSX example.)

```bash
# start somewhere outside a git repo.
git clone https://github.com/QCMM/iodata.git
# rewire the remotes
git remote rename origin qcmm
git remote add origin git@github.com:theochem/iodata.git
# rename your development branch
git branch -m master orca
# get back the master branch from theochem/iodata
git fetch origin master:master
# run the linters
./setup.py build_ext -i
cardboardlinter -r master
# You can commit fixes now.
# force-push the master to your repo
git push qcmm master:master -f
# Now you can close this PR.
# Time to push the feature branch to your fork:
git push qcmm orca
# You should get a link on your terminal to make a PR for this branch.
```

File: bin/horton-convert [hunk 1]
```diff
@@ -36,7 +36,7 @@ def parse_args():
                                                  ' This only works if the input contains sufficient'
                                                  ' data for the output')
     parser.add_argument('-V', '--version', action='version',
-                        version="%%(prog)s (HORTON version %s)" % version.__version__)
+                        version="%%(prog)s (HORTON version 2.0)" )
```
Comment (matt-chan) [inline bin/horton-convert]: We used this in the past so you could define version numbers using git tags. It was too easy to neglect to update a line somewhere and have inconsistent version numbers in the program. 

The downside is that if you didn't use the travisCI infrastructure, the version.py file didn't get generated. It's probably better to use a `try: ... except ImportError ...` fallback to get around this instead of hard coding the number.

Comment (tovrstra) [reply bin/horton-convert]: Yeah, we should make this easier. I'm working on something (roberto), but is not quite ready yet.

Comment (evohringer) [reply bin/horton-convert]: I did add a try: ... except ImporError as suggested by Matt. Hope this is fine. Thanks @matt-chan

**Comment by evohringer** [comment]
> That probably is probably because you used the `master` branch (and also `origin/master`) for your development. Do you still have a reference in your repo to the master branch of `theochem/iodata`? That would be needed to get the same test output locally.
> 
> It is in general a good idea to use the master branch of cloned repos only to follow up the upstream commits. For clarity, it is recommended to make a new "feature" branch in which you make your changes. This way, you can always easily compare with what you started from, or update the master branch with new upstream developments.
> 
> This is a quick summary of that workflow, assuming you would start from scratch, just to sketch the idea:
> 
> ```shell
> # Clone the primary repo first
> git clone git@github.com:theochem/iodata.git
> # Now `origin` refers to theochem/iodata
> # Add your fork as the second remote (after making the fork on github.com)
> cd iodata
> git remote add qcmm git@github.com:qcmm/iodata.git
> # Make a feature branch, in which a new feature will be developed
> git checkout -b orca
> # After some changes commit these:
> git add ...
> git commit ...
> # Push the feature branch to  your repo
> git push qcmm feature
> # A link will be printed on your terminal to make a pull request
> # If needed make more commits in the feature branch and push them to qcmm/iodata.
> # The pull request will be updated automatically.
> ```
> 
> With this setup `origin/master` still refers to the one from `theochem/iodata`.
> 
> You can check your current remotes as follows, with the output I typically get as comments:
> 
> ```shell
> git remote show
> # origin
> # tovrstra
> git remote show origin
> #* remote origin
> #  Fetch URL: git@github.com:theochem/iodata.git
> #  Push  URL: git@github.com:theochem/iodata.git
> #  HEAD branch: master
> #  Remote branches:
> #    master            tracked
> #  Local branch configured for 'git pull':
> #    master rebases onto remote master
> #  Local ref configured for 'git push':
> #    master pushes to master (up to date)
> git remote show tovrstra
> #* remote tovrstra
> #  Fetch URL: git@github.com:tovrstra/iodata.git
> #  Push  URL: git@github.com:tovrstra/iodata.git
> #  HEAD branch: master
> #  Remote branches:
> #    master  new (next fetch will store in remotes/tovrstra)
> #  Local refs configured for 'git push':
> #    master pushes to master (fast-forwardable)
> ```
> 
> (I removed some branches for clarity.)
> With this setup, I only pull commits from `origin` (short name for `theochem/iodata`), never push to it, so I don't even need write access to it. It can pull and push with the second remote, i.e. `tovrstra`. I guess you now have
> 
> ```shell
> git remote show origin
> #* remote origin
> #  Fetch URL: git://github.com/QCMM/iodata.git
> #  Push  URL: git://github.com/QCMM/iodata.git
> #  HEAD branch: master
> #  Remote branches:
> #    master            tracked
> #  Local ref configured for 'git push':
> #    master pushes to master (local out of date)
> ```
> 
> The following commands will show how to switch in a new clone of your fork. A verbose command prompt helps a lot to see on which branch you are working. (See e.g. https://gist.github.com/kevin-smets/8568070 for an OSX example.)
> 
> ```shell
> # start somewhere outside a git repo.
> git clone https://github.com/QCMM/iodata.git
> # rewire the remotes
> git remote rename origin qcmm
> git remote add origin git@github.com:theochem/iodata.git
> # rename your development branch
> git branch -m master orca
> # get back the master branch from theochem/iodata
> git fetch origin master:master
> # run the linters
> ./setup.py build_ext -i
> cardboardlinter -r master
> # You can commit fixes now.
> # force-push the master to your repo
> git push qcmm master:master -f
> # Now you can close this PR.
> # Time to push the feature branch to your fork:
> git push qcmm orca
> # You should get a link on your terminal to make a PR for this branch.
> ```

Thanks Toon for the nice guide. If this is ok I would try it for the next implementation and then I try from scratch as you suggested. 

One thing which is bothering me is that installing IOData locally does not allow me to run the tests in the installed version because the "__path__" of iodata is not set. I guess this is done automatically in travis right?? Is there a way to make two steps locally before submitting:
1.) run nosetests 
2.) run cardboarlint (for this I now know how to do that and will do it for the next implementation)

Sorry to bother with this stupid things. I hope that conversation also help others implementing new features in iodata.

**Comment by tovrstra** [comment]
Sure, no problem.

I don't fully understand your question regarding the path, but I'll try to give some answer. At the moment, the following steps are needed to run the tests in the source tree. The instructions are given for a fresh clone of the repo, which I just tested:

```bash
git clone git@github.com:theochem/iodata.git
cd iodata
python3 tools/gitversion.py python > iodata/version.py
python3 -m pip install pytest cython --user 
python3 setup.py build_ext -i
python3 -m pytest iodata
```

I agree, this is not intuitive at all, and this should become easier, which I'm working on. Let me clarify a few things, so you can get around it until we have a better way:

- On OSX, python scripts installed with pip with the `--user` option, tend to get installed in a directory that is not present in the PATH variable. You either have to modify `PATH` to fix this or you can run most programs as `python -m [name_of_module] [args]`. I used that trick above to increase the chances that it will work on OSX. I don't know which directory you would have to add to the PATH because I have no access to OSX. This is an example of that problem, but not sure if it is representative: https://travis-ci.com/theochem/roberto/jobs/183614464#L103 (line 103)

- If python3 is already the default python on your system, you may just use `python` instead of `python3`.

- We recently started using pytest instead of nosetests. It is a more up-to-date testing framework compared to nosetests.

P.S. I'll take the quote of my message out of your post for clarity.

File: .cardboardlint.yml~ [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+- import:
+    packages: ['iodata']
+- namespace:
+    filefilter: ['- */__init__.py', '- */test_*.py', '- *setup.py', '- tools/*',
+                 '+ *.py', '+ *.pyx']
+- pylint:
+- pycodestyle:
+    config: .pycodestylerc
+- pydocstyle:
+- whitespace:
```
Comment (tovrstra) [inline .cardboardlint.yml~]: Can you remove this file? I think it is a backup copy that accidentally committed.

File: bin/horton-convert [hunk 3]
```diff
@@ -36,15 +36,18 @@ def parse_args():
                                                  ' This only works if the input contains sufficient'
                                                  ' data for the output')
     parser.add_argument('-V', '--version', action='version',
-                        version="%%(prog)s (HORTON version %s)" % version.__version__)
-
+                        try:
+                            version="%%(prog)s (HORTON version %s)" % version.__version__
+                        except ImportError:
+                            raise ImportError('No version.py file found'))
```
Comment (tovrstra) [inline bin/horton-convert]: version.py should normally be present. (See my last post on how to add it.)

File: iodata/orca.py [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+    Parameters
+    ----------
+    f: TextIO
+       A ORCA file object (in read mode).
+
+    Returns
+    -------
+    out: int
+       Total number of atoms.
```
Comment (tovrstra) [inline iodata/orca.py]: This is a really minor comment, but could be useful. It is typically convenient to name the return variables the same is in the code.

File: iodata/test/test_orca.py [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+try:
+    from importlib_resources import path
+except ImportError:
+    from importlib.resources import path
+
+
+def test_load_water_number():
+    # test if IOData has atomic numbers
+    with path('iodata.test.data', 'water_orca.out') as fn_xyz:
+        mol = IOData.from_file(str(fn_xyz))
```
Comment (tovrstra) [inline iodata/test/test_orca.py]: `str` function may not be needed. I've seen it before but normally path objects get converted to strings when needed, safe for some corner cases maybe?

File: iodata/test/test_orca.py [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+def check_water(mol):
+    """Checks if atomic numbers and coordinates obtained from orca out file are correct.
+
+        Parameters
+        ----------
+        mol : IOData
+            IOdata dictionary.
+
+        Returns
```
Comment (tovrstra) [inline iodata/test/test_orca.py]: No need to document return if nothing is returned.

File: iodata/test/test_orca.py [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+        -------
+        None
+    """
+    assert mol.numbers[0] == 8
+    assert mol.numbers[1] == 1
+    assert mol.numbers[2] == 1
```
Comment (tovrstra) [inline iodata/test/test_orca.py]: Again a very minor suggestion, mainly sharing it because you might like the trick:

```python
# The following line does not start with assert, because this is done inside the assert_equal function.
np.testing.assert_equal(mol.numbers, [8, 1, 1])
```

The module `np.testing` contains a lot of convenient tricks to write assertions for arrays. See https://docs.scipy.org/doc/numpy-1.13.0/reference/routines.testing.html

File: iodata/test/test_orca.py [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+    # test if IOData has the correct energy
+    with path('iodata.test.data', 'water_orca.out') as fn_xyz:
+        mol = IOData.from_file(str(fn_xyz))
+    check_water(mol)
+
+
+def check_water(mol):
+    """Checks if atomic numbers and coordinates obtained from orca out file are correct.
+
+        Parameters
```
Comment (tovrstra) [inline iodata/test/test_orca.py]: Can you remove one space of indentation from this and following lines in the docstring. (travis tests fail on this)

File: iodata/test/test_orca.py [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+        ----------
+        mol : IOData
+            IOdata dictionary.
+
+        Returns
+        -------
+        None
+    """
```
Comment (tovrstra) [inline iodata/test/test_orca.py]: Can you add a blank line after the last section of a docstring? (also from `pycodestyle`)

File: .cardboardlint.yml~ (hunk 2 above)
Comment (evohringer) [reply .cardboardlint.yml~]: done

File: bin/horton-convert (hunk 3 above)
Comment (evohringer) [reply bin/horton-convert]: If this is ok I will leave for people who just try it out after installation

File: iodata/orca.py (hunk 4 above)
Comment (evohringer) [reply iodata/orca.py]: changed to natom

File: iodata/test/test_orca.py (hunk 5 above)
Comment (evohringer) [reply iodata/test/test_orca.py]: deleted str

File: iodata/test/test_orca.py (hunk 6 above)
Comment (evohringer) [reply iodata/test/test_orca.py]: deleted Return

File: iodata/test/test_orca.py (hunk 7 above)
Comment (evohringer) [reply iodata/test/test_orca.py]: Changed to np.testing. This is a nice feature. I will have a look.

File: iodata/test/test_orca.py (hunk 8 above)
Comment (evohringer) [reply iodata/test/test_orca.py]: done

File: iodata/test/test_orca.py (hunk 9 above)
Comment (evohringer) [reply iodata/test/test_orca.py]: done

**Comment by FarnazH** [comment]
A few small things:
**1)** Can you please use `iodata` HEADER for `orca.py` and `horton-convert`. It can be found here: https://github.com/theochem/iodata/blob/master/HEADER
**2)** Probably `horton-convert` should be renamed as `iodata-convert`.

File: iodata/test/test_orca.py (hunk 8 above)
Comment (FarnazH) [reply iodata/test/test_orca.py]: Can you please replace `Checks` with `Check`? First line should be in imperative mood.

File: bin/horton-convert (hunk 3 above)
Comment (tovrstra) [reply bin/horton-convert]: ok, until our build and development workflow is simplified, we can have this. It is convenient.

**Comment by evohringer** [comment]
@to

> A few small things:
> **1)** Can you please use `iodata` HEADER for `orca.py` and `horton-convert`. It can be found here: https://github.com/theochem/iodata/blob/master/HEADER
> **2)** Probably `horton-convert` should be renamed as `iodata-convert`.

All done. Thanks for checking.

**Comment by tovrstra** [comment]
LGTM. Thanks!

**Comment by evohringer** [comment]
No green button to merge. Everything is green but I can not find the button.

The last part says: 
This branch has no conflicts with the base branch
Only those with write access to this repository can merge pull requests.

**Comment by tovrstra** [comment]
OK that's right, only few have write access to avoid unfortunate accidents. I'll do it.
